ui/src/App.scss

```diff
@@ -1,16 +1,27 @@
 .app {
-  width: 100%;
-  height: 100%;
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
   background-color: #eecbcb;
   overflow-y: auto;
```

Comment:
Get rid of this line, we no longer need it since `html {overflow-y: scroll}` does what we want, good job. I coudlnt even figure out this behavior